gst/nnstreamer/tensor_common.c

```diff
@@ -856,6 +856,35 @@ gst_tensors_config_from_structure (GstTensorsConfig * config,
   return TRUE;
 }
 
+/**
+ * @brief Parse caps from peer pad and set tensors config.
+ * @param pad GstPad to get the capabilities
+ * @param config tensors config structure to be filled
+ * @param is_fixed flag to be updated when peer caps is fixed (not mandatory, do nothing when the param is null)
+ * @return TRUE if peer has tensor caps
```

Comment:
I think your intention is `If the tensors config successfully obtained from the peers`.
This sentence is confusing because of `gst_pad_peer_has_tensor_caps`.
This is returned TRUE if peer has tensor caps.